.github/workflows/kvrocks.yaml

```diff
@@ -310,7 +310,18 @@ jobs:
           tags: kvrocks:ci
           outputs: type=docker
       - name: Test built image
-        run: docker run --rm kvrocks:ci -v
```

Comment:
I think we still need this one since it is not covered in the new checks.

Reply:
When starting kvrocks, the version is output but cannot be seen due to detach. 
I have restored this version check to output the version to github action log.